engine/src/main/java/org/terasology/rendering/dag/nodes/DeferredPointLightsNode.java

```diff
@@ -92,7 +85,12 @@ public void initialise() {
 
         addDesiredStateChange(new DisableDepthTest());
 
+        addDesiredStateChange(new BindFBO(READONLY_GBUFFER, displayResolutionDependentFBOs));
```

Comment:
Not really. I'll add it for now for the sake of consistency, but since the default viewPort is the size of screen, I would recommend that we remove all setViewportToSizeOf(READONLY_GBUFFER) StateChanges, seeing how redundant they are.